test/index.test.js

```diff
@@ -272,7 +272,7 @@ describe('Get project by type', () => {
         if (err) return done(err);
         const { message } = res.body;
         expect(message).toEqual([
-          'projectType must be one of the following values: internal, remotely',
+          'projectType must be one of the following values: internal, remotely, INTERNAL, REMOTELY',
```

Comment:
@MohammedAlghazali Why are both lowercase and uppercase needed?  Maybe the reason could be documented for future developers with a comment here.

Reply:
oh, I have confused with this, I thought we made it all as lowercase (using toLowerCase()) when we add it to the database But we did not, so I will modify it as previous and will make it lowercase in (onFinsh) before I send it to the server.

Reply:
Sounds good! Better to clean it up before putting it into the database than having a bunch of different capitalizations of the same value in the database!